lib/factory_bot/decorator.rb

```diff
@@ -6,18 +6,30 @@ def initialize(component)
       @component = component
     end
 
-    def method_missing(name, *args, &block) # rubocop:disable Style/MethodMissingSuper
-      @component.send(name, *args, &block)
+    if ::Gem::Version.new(::RUBY_VERSION) >= ::Gem::Version.new("2.7")
+      class_eval(<<~RUBY, __FILE__, __LINE__ + 1)
+          def method_missing(...) # rubocop:disable Style/MethodMissingSuper, Style/MissingRespondToMissing
```

Comment:
We switched over from Rubocop to [standard](https://github.com/testdouble/standard), so feel free to remove these Rubocop directives.

Reply:
Standard does seem to still use the `rubocop:disable` comments since removing these (maybe not this one since it is `class_eval`ed) causes the standard test when I run `bundle exec rake` to fail.